### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by MinhasMA**

### CODE REVIEW

**Comment by selsta** [comment]
Any reason for this change?

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Multistage docker build, requires docker 17.05
 
 # builder stage
-FROM ubuntu:20.04 as builder
+FROM ubuntu:latest as builder
```
Comment (mj-xmr) [inline Dockerfile:4]: It sounds like a good idea for the version to be self-adjusting across time, but I'd prefer having the version fixed in stone, until there's somebody, who updates it to the next latest version, and collects the medal for fixing all the resulting incompatibilities.

Your proposed change would strike you when you're least expecting it.

File: Dockerfile [hunk 2]
```diff
@@ -19,7 +19,7 @@ RUN set -ex && \
         pkg-config \
         gperf
 
-WORKDIR /src
+WORKDIR /src/1
```
Comment (mj-xmr) [inline Dockerfile:22]: Could you explain this?
Did you mean something like `/src/temp` ?

File: Dockerfile (hunk 1 above)
Comment (ndorf) [reply Dockerfile:4]: I agree, the base should be a known-good version.

**Comment by selsta** [comment]
Closing as the author did not reply.

**Pull request closed without merging**